native/src/main/java/io/ballerina/lib/pdfbox/utils/Utils.java

```diff
@@ -0,0 +1,14 @@
+package io.ballerina.lib.pdfbox.utils;
+
+import io.ballerina.runtime.api.creators.ErrorCreator;
+import io.ballerina.runtime.api.utils.StringUtils;
+import io.ballerina.runtime.api.values.BError;
+
+public class Utils {
+
+    public static BError getBError(String message) {
```

Comment:
Let's update this method to accept `Throwable` instance as well.
`public static BError getBError(String message, Throwable throwable)`

When generating the `BError`, you can now pass the error message along with the error cause as well. (instead of adding it as null)

Refer:
https://github.com/ballerina-platform/module-ballerina-avro/blob/main/native/src/main/java/io/ballerina/lib/avro/Utils.java#L41

Reply:
done!